.travis.yml

```diff
@@ -22,6 +22,9 @@ matrix:
 
 before_install:
   - docker run -d -p 5984:5984 --net=host --name couch apache/couchdb:2.3
+  - curl -sL https://deb.nodesource.com/setup_8.x | sudo bash -
+  - sudo apt install nodejs
+  - sudo npm install -g eslint@5.16.0 prettier eslint-plugin-prettier eslint-config-cozy-app
```

Comment:
Can we move this line to `scripts/lint.sh` (without `sudo`), as it is only for lint and not for the other builds on travis?